pkgs/applications/misc/survex/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, autoreconfHook
+, pkg-config
+, wxGTK30-gtk3
+, ffmpeg
+, proj
+, perl532
+, unscii
+, python
+, libGL
+, libGLU
+, x11
+, docbook2x
+, docbook5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "survex";
+  version = "1.2.44";
+
+  nativeBuildInputs = [ docbook5 x11 libGL libGLU docbook2x autoreconfHook pkg-config wxGTK30-gtk3 ffmpeg proj python (perl532.withPackages (perlPackages: with perlPackages; [ LocalePO ] )) ];
+
+  src = builtins.fetchGit {
+    url = "git://git.survex.com/survex";
+    rev = "38bc41554efda8c15494ca44e5fc17c8146a077f";
+    ref = "refs/tags/${version}";
+  };
+
+  # Docs rely on sgmltools-lite, a package that would be quite complex to
+  # provide as it is quite old. So this preConfigure hook effectively disables
+  # the doc generation. An example of packaging sgmltools-lite from Gentoo can
+  # be found here:
+  # https://gitweb.gentoo.org/repo/gentoo.git/tree/app-text/sgmltools-lite/sgmltools-lite-3.0.3-r15.ebuild?id=0b8b716331049599ea3299981e3a9ea6e258c5e0
+
+  preConfigure = ''
+    echo "" > doc/Makefile.am
+  '';
+
+  postConfigure = ''
+#    substituteInPlace doc/Makefile --replace "docbook2man" "docbook2man --sgml" # Will be needed once sgmltools-lite is packaged.
```

Comment:
```suggestion
```

Reply:
Why would you remove the comment? It explains the state of the doc being compiled. Is it considered bad to have comments in Nixpkgs?

Reply:
I think comments are fine, but comments that refer to unpackaged dependencies and possible problems with them are IMO kind of pointless unless you also plan to package that dependency eventually.

If you want to leave it in, then please fix the indentation at least.

Reply:
The indentation should be just two spaces, correct?

Reply:
Should be:

```suggestion
    # substituteInPlace doc/Makefile --replace "docbook2man" "docbook2man --sgml" # Will be needed once sgmltools-lite is packaged.
```